packages/yuudachi/src/functions/anti-raid/blastOff.ts

```diff
@@ -70,13 +78,12 @@ export async function blastOff(
 					action: CaseAction.Ban,
 					multi: true,
 				}),
-				args.dryRun,
 			)
 				.then((case_) => {
 					result.push({ member, success: true });
 					return case_;
 				})
-				.catch(() => result.push({ member, success: false }))
+				.catch((err) => result.push({ member, success: false, error: (err as Error).message }))
```

Comment:
No need to forward the error here. We aren't doing anything with it.

Reply:
The error will be used in the report when ready, but I can leave this forwarding to that PR

Reply:
Yeah right now its not doing anything, so its kinda pointless